sql/service/src/main/java/org/apache/spark/sql/service/cli/session/ServiceSessionImpl.java

```diff
@@ -64,8 +64,6 @@
   private File sessionLogDir;
   private volatile long lastAccessTime;
   private volatile long lastIdleTime;
-  private Map<String, String> sessionVariables = new HashMap<String, String>();
-  private Map<String, String> sessionOverriddenConf = new HashMap<String, String>();
```

Comment:
Those were not used by Spark and that's why they're removed, yes?

Reply:
> Those were not used by Spark and that's why they're removed, yes?

Session Conf and Session Var can be just used by `sqlConf.setConf`, and that's it's the way current Spark ThriftServer do. So we can just use that way to replace and remove redundant code